workspaces/vscode-python/Dockerfile

```diff
@@ -0,0 +1,140 @@
+# Enhanced code-server workspace template for PrairieLearn.
+# As discussed here: https://github.com/PrairieLearn/PrairieLearn/issues/3170
+
+FROM --platform=linux/amd64 codercom/code-server:4.20.1-bookworm
+
+# On PL, we want to standardize on using 1001:1001 for the user. We change the
+# "coder" account's UID and GID here so that fixuid has no work to do later.
+# This speeds up container loading time drastically and avoids timeouts.
+USER root
+RUN export OLD_UID=$(id -u coder) && \
+	export OLD_GID=$(id -g coder) && \
+	export NEW_UID=1001 && \
+	export NEW_GID=1001 && \
+	groupmod -g 1001 coder && \
+	usermod -u 1001 -g 1001 coder && \
+	find /home -user $OLD_UID -execdir chown -h $NEW_UID {} + && \
+	find /home -group $OLD_GID -execdir chgrp -h $NEW_GID {} + && \
+	unset OLD_UID OLD_GID NEW_UID NEW_GID
+	
+# Install Python via miniconda
+USER coder
+RUN curl -sfLO https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+ && /bin/bash Miniconda3-latest-Linux-x86_64.sh -b -p /home/coder/miniconda \
+ && PATH="/home/coder/miniconda/bin:$PATH" \
+ && /home/coder/miniconda/condabin/conda init bash \
+ && conda config --add channels conda-forge \
+ && conda install ipython \
+ && conda install pandas \ 
+ && conda install seaborn \
+ && conda install pillow
+
+# Cleaning up the cache in the same step when we install is important if you
+# want to minimize the image size.
+USER coder
+RUN ( code-server --disable-telemetry --force \
+		# Install only one extension per line.
+		--install-extension ms-python.python \
+		--install-extension njpwerner.autodocstring \
+		# --install-extension ms-python.debugpy \ # Cannot find the debugger.
```

Comment:
@Victorsss-Orz how did you find the Default VS Code debugger? 

There is another [Python Debugger](https://marketplace.visualstudio.com/items?itemName=ms-python.debugpy) that Microsoft has, but I don't think it's available in the "Open Source" extension library so we would need to add the .vsix file if we want to use it.

Otherwise, if you think the default VS Code debugger is fine, maybe we leave that one there as-is.

Reply:
I've never used other debuggers than the one included in VSCode. The current one supports toggling breakpoints and seeing the variable values between breakpoints. You can just use the "run" tab at the left of the workspace, and choose to run with or without debugging.

Reply:
Okay sounds good. You can remove that contented line then